__tests__/components/forecast-summaries.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummaries from '../../src/components/forecast-summaries';
+import ForecastSummary from '../../src/components/forecast-summary';
+
+const forecasts = [
+  {
+    date: 123,
+    description: 'date1',
+    icon: 'icon1',
+    temperature: {
+      max: 999,
+    },
+  },
+  {
+    date: 456,
+    description: 'date2',
+    icon: 'icon2',
+    temperature: {
+      max: 777,
+    },
+  },
+];
+it('renders the correct amount of ForecastSummary components', () => {
+  const wrapper = Enzyme.shallow((
+    <ForecastSummaries forecasts={forecasts} />
+  ));
+  expect(wrapper.find(ForecastSummary).length).toEqual(2);
```

Comment:
Try to avoid using `magic numbers` in you code. The two here won't mean anything to other people who could be reading this. `forecasts.length` gives a lot more information.